src/main/java/com/winner_is_kungen/tda367/model/LogicGates/Input.java

```diff
@@ -0,0 +1,41 @@
+package com.winner_is_kungen.tda367.model.LogicGates;
+
+import com.winner_is_kungen.tda367.model.Component;
+import com.winner_is_kungen.tda367.model.ComponentListener;
+import com.winner_is_kungen.tda367.model.util.Tuple;
+
+public class Input extends Component {
+	private static final String typeID = "INPUT";
+	private boolean state = false;
+
+	/**
+	 * Constructor for input-component.
+	 *
+	 * @param id unique String specific to each instance of Input.
+	 */
+	public Input(String id) {
+		super(id, typeID, 0, 1);
+	}
+
+	//Getters
+
+	public boolean getState() {
+		return state;
+	}
+
+	public static String getTypeID() {
+		return typeID;
+	}
```

Comment:
Good idea making a getter. We should probably have getters for the type ID of the other components too.

But we shouldn't mix instance and static methods, move it above the `getState` method.